modules/custom/view_count/view_count.module

```diff
@@ -2,20 +2,30 @@
 
 use Drupal\Component\Utility\UrlHelper;
 use Drupal\node\Entity\Node;
+use Drupal\user\Entity\User;
 use Drupal\view_count\GetViewDataService;
 
-function view_count_node_view(
-  array &$build, \Drupal\Core\Entity\EntityInterface $entity, \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display, $view_mode) {
+/**
+ * @param array $build
+ * @param \Drupal\Core\Entity\EntityInterface $entity
+ * @param \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display
+ * @param $view_mode
+ */
+function view_count_node_view(array &$build, \Drupal\Core\Entity\EntityInterface $entity, \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display, $view_mode) {
   if ($view_mode === 'full') {
-
-    // Get data view count report input.
     $nid = \Drupal::routeMatch()->getRawParameter('node');
 
-    $view_results = new GetViewDataService;
+    // Get data for report on node.
+    $view_results = \Drupal::service('view_count.get_view_data_service');
```

Comment:
I would rename it to something indicating it's a service. Like views_count_service